fe/fe-core/src/main/java/com/starrocks/catalog/TabletStatMgr.java

```diff
@@ -132,7 +132,11 @@ private void updateTabletStat(Long beId, TTabletStatResult result) {
                 continue;
             }
             // TODO(cmy) no db lock protected. I think it is ok even we get wrong row num
-            replica.updateStat(entry.getValue().getData_size(), entry.getValue().getRow_num());
+            replica.updateStat(
```

Comment:
Since you have update the version count in the TabletStatMgr. Is it necessary to delete the code that update the version count in tablet report?